README.md

```diff
@@ -213,6 +315,163 @@ module Maintenance
 end
 ```
 
+### Writing test for a custom Task
+
+As with other tasks, you should write tests for your `#process` method. It will
+receive the first item in each pair your custom Enumerator yields (`[item,
+item_cursor]`).
+
+You should also ensure your Enumerator is tested, by unit testing it in
+isolation, testing your `#enumerator_builder` method, or both. Make sure you
+test how your Enumerator handles the absence or presence of a `context.cursor`,
```

Comment:
I think this is the main advice we should give them about testing their EnumeratorBuilder.